openapi/openapi.json

```diff
@@ -25791,6 +25791,20 @@
       "description": "AWSCSIDriverConfigSpec defines properties that can be configured for the AWS CSI driver.",
       "type": "object",
       "properties": {
+        "efsVolumeMetricsFSRateLimit": {
+          "description": "efsVolumeMetricsFSRateLimit defines the rate limit, in goroutines per file system, for processing volume metrics. Requires efsVolumeMetricsOptIn to be enabled. The default rate limit set by the EFS CSI driver is 240 goroutines per file system.",
+          "type": "integer",
+          "format": "int32"
+        },
+        "efsVolumeMetricsOptIn": {
+          "description": "efsVolumeMetricsOptIn enables metrics for EFS CSI Driver volumes. When enabled, the CSI driver traverses the entire volume to calculate and return metrics. This operation can significantly impact system performance due to the extensive volume walk required. Use with caution in performance-sensitive environments.",
```

Comment:
```suggestion
          "description": "efsVolumeMetricsOptIn enables metrics for EFS CSI Driver volumes. When enabled, the CSI driver traverses the entire volume to calculate and return volume usage metrics. This operation can significantly impact system performance due to the extensive volume walk required. Use with caution in performance-sensitive environments.",
```

Reply:
I would even add names of the metrics.

Reply:
I am reluctant to include metric names because upstream changes—such as adding, removing, or modifying existing names—could lead to outdated documentation?

Reply:
Ack.

LGTM from storage point of view